src/main/java/com/bathanh/exercise/oop/oop1/Rectangle.java

```diff
@@ -0,0 +1,42 @@
+package com.bathanh.exercise.oop.oop1;
+
+public class Rectangle implements Shape {
+
+    private final Point pointBottomLeft;
+    private final Point pointTopRight;
```

Comment:
Không nên dùng prefix, postfix

```suggestion
    private final Point bottomLeft;
    private final Point topRight;
```

Reply:
resolved